### ISSUE TITLE
Hide Marketplace Ads

### ISSUE THREAD

**Issue by TheOneThatOofs** [AllanWang/Frost-for-Facebook#1635]
I would like a way to get rid of those pesky marketplace ads, especially when I have no history of searching such things. It appears to be a mobile site issue, because I compared it with my mobile browser and got the same results. I have started seeing these since two days ago. I can't even hide them through Facebook normally, because the 3-dot menu cannot be interacted with. There seems to be no such bug reported through my searches on Google so far. Attached is the Frost screenshot (in darkmode) and a browser one from Firefox with an adblocker (ghostery).
![Screenshot_20200208-083118](https://user-images.githubusercontent.com/60817046/74086634-b30d7a00-4a52-11ea-9296-82352b33f848.png)
![Screenshot_20200208-085125](https://user-images.githubusercontent.com/60817046/74086636-ba348800-4a52-11ea-8a31-e540a8ffc220.png)

**Comment by AllanWang** [comment]
Are these listings related to your marketplace search results? If it's trivial, I may try to add a filter, but if they are moreso "related searches" I will not be bundling it with the ad blocker.

**Comment by TheOneThatOofs** [comment]
No, I never use marketplace or even searched for such things. They just keep clogging up my feed with almost six of the same posts consecutively.

**Comment by AllanWang** [comment]
Oh I see, it's in your newsfeed. I will remove it if I can find something similar on my feed
